IPs/RSKIP185.md

```diff
@@ -0,0 +1,70 @@
+# Title
+
+|RSKIP          |185           |
+| :------------ |:-------------|
+|**Title**      |Peg-out refund and events |
+|**Created**    |19-NOV-20 |
+|**Author**     |JD |
+|**Purpose**    |Usa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+The Bridge offers a protocol to convert R-BTC back in BTC. This functionality is known as a peg-out.
+To do this a user has to send R-BTC to the Bridge and it will start the operation.
+
+This RSKIP proposes improvements in this protocol to refund users that send less than the minimum required amount and at the same time proposes to create events to easily known which operations were successful and which ones weren't.
+
+## Motivation
+
+As the platform gains traction this operation will be more common and users mistakes become more probable, therefore we consider important improve the protocol to avoid money loss.
```

Comment:
```suggestion
As the platform gains traction, this operation will be more common and users' mistakes become more probable. Therefore we consider it important to improve the protocol to avoid money losses.
```